cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -1354,3 +1354,175 @@ def test_update_committee_threshold_out_of_range(
+            prev_action_ix=prev_action_rec.ix,
+            deposit_return_stake_vkey_file=pool_user_lg.stake.vkey_file,
+        )
+
+        tx_files = clusterlib.TxFiles(
+            proposal_files=[update_threshold_action.action_file],
+            signing_key_files=[
+                pool_user_lg.payment.skey_file,
+            ],
+        )
+
+        tx_output_action = clusterlib_utils.build_and_submit_tx(
+            cluster_obj=cluster,
+            name_template=f"{temp_template}_update_threshold",
+            src_address=pool_user_lg.payment.address,
+            use_build_cmd=True,
+            tx_files=tx_files,
+        )
+
+        action_txid = cluster.g_transaction.get_txid(tx_body_file=tx_output_action.out_file)
+        action_gov_state = cluster.g_conway_governance.query.gov_state()
+        _cur_epoch = cluster.g_query.get_epoch()
+        conway_common.save_gov_state(
+            gov_state=action_gov_state, name_template=f"{temp_template}_action_{_cur_epoch}"
+        )
+        prop_action = governance_utils.lookup_proposal(
+            gov_state=action_gov_state, action_txid=action_txid
+        )
+        action_ix = prop_action["actionId"]["govActionIx"]
+
+        # Make sure the votes don't happen close to epoch boundary
+        clusterlib_utils.wait_for_epoch_interval(
+            cluster_obj=cluster, start=10, stop=common.EPOCH_STOP_SEC_BUFFER
+        )
+
+        conway_common.cast_vote(
+            cluster_obj=cluster,
+            governance_data=governance_data,
+            name_template=f"{temp_template}_add_yes",
+            payment_addr=pool_user_lg.payment,
+            action_txid=action_txid,
+            action_ix=action_ix,
+            approve_drep=True,
+            approve_spo=True,
+        )
+
+        _check_rat_enact_state(
+            action_txid=action_txid,
+            action_type=governance_utils.PrevGovActionIds.COMMITTEE,
+        )
+
+        # Try to ratify a "create constitution" action that is expecting approval from the CC
+        anchor_url = "http://www.const-action.com"
+        constitution_url = "http://www.const-new.com"
+        (
+            const_action,
+            const_action_txid,
+            const_action_ix,
+        ) = conway_common.propose_change_constitution(
+            cluster_obj=cluster,
+            name_template=f"{temp_template}_constitution",
+            anchor_url=anchor_url,
+            anchor_data_hash=cluster.g_conway_governance.get_anchor_data_hash(text=anchor_url),
+            constitution_url=constitution_url,
+            constitution_hash=cluster.g_conway_governance.get_anchor_data_hash(
+                text=constitution_url
+            ),
+            pool_user=pool_user_lg,
+        )
+
+        # Make sure the votes don't happen close to epoch boundary
+        clusterlib_utils.wait_for_epoch_interval(
+            cluster_obj=cluster, start=10, stop=common.EPOCH_STOP_SEC_BUFFER
+        )
```

Comment:
Move this before submitting the proposal. We have only two epochs before the proposal expires, so we better submit AND approve the proposal in the same epoch.